README.md

```diff
@@ -19,6 +19,7 @@ FIREHOSE_AWS_REGION=us-west-2 \
 FIREHOSE_STREAM_NAME=firehose-test \
 LOG_FILE=/tmp/kcl_stderr \
 RATE_LIMIT=100 \
+DEPLOY_ENV="local" \
```

Comment:
`_DEPLOY_ENV` is injected by Catapult -- It should be equivalent to use that instead, it just puts the burden on Catapult instead of the app. 

In the past I think we've been uncertain as to whether app code should depend on Catapult injected env vars.